spring-grpc-docs/src/main/antora/modules/ROOT/pages/client.adoc

```diff
@@ -131,3 +131,8 @@ However, by setting the `mergeWithGlobalInterceptors` parameter on the channel f
 You can use this option if you want to add a per-client interceptor between global interceptors.
 
 IMPORTANT: The per-channel interceptors you pass in must either be bean instances marked with `@Order` or regular objects that implement the `Ordered` interface to be properly merged/ordered with the global interceptors.
+
+== Observability
+
+Spring gRPC provides an autoconfigured interceptor that can be used to provide observability to your gRPC services.
+All you need to do is add Spring Boot actuators to your project, and optionally a bridge to your observability platform of choice (just like https://docs.spring.io/spring-boot/reference/actuator/observability.html[any other Spring Boot application]).
```

Comment:
Would actuators be necessary or appropriate for a client app (no server)?

Reply:
Yes I agree, I can remove the part talking about Spring Boot actuators and leave only the first line.

If you have more feedback on what else should be stated in the docs please let me know. Writing docs is not my strong side 😅